js_src/toggle.ts

```diff
@@ -0,0 +1,61 @@
+const toggleProperties = {
+    /* "display" is for toggling, whether an element is displayed or not */
+    display: {
+        getValue: (el, value) => el.style.display,
+        toggle: (el, value, defaultValue) => {
+            el.style.display = (el.style.display == "none") ? defaultValue : "none";
+        }
+    },
+
+    /* "class" is for toggling, whether an element has a specified class or not */
+    class: {
+        getValue: (el, value) => { value.split(" ").map((i) => [i, el.classList.contains(i)]) },
```

Comment:
`i` isnt a great variable name here, lets change it to `className`. I'd also probably be inclined to split this up across some lines as its nested.